exercises/bracket-push/Cargo.toml

```diff
@@ -1,3 +1,3 @@
 [package]
 name = "bracket-push"
-version = "1.2.0"
+version = "1.3.0"
```

Comment:
this should not change until the test cases match https://github.com/exercism/problem-specifications/blob/master/exercises/bracket-push/canonical-data.json 1.3.0. It should go in a separate PR.

Reply:
Ah ok, so that's what it's used for.